package.json

```diff
@@ -77,7 +77,9 @@
     "web3": "^1.10.0"
   },
   "dependencies": {
+    "@types/jest": "^29.5.7",
```

Comment:
Shouldn't this be in the dev dependencies? This might increase the size of the package and not necessary to ship with the package.
@Hugoo ?

Reply:
Moving it, makes sense.

Reply:
FYI: jest was also missing, causing the repo to assume jest from global. The current version of jest is 29 which is most of the time not compatible with 27.

Reply:
Moved.